File: part 6.5/loop5.py
# 5. Bintang asteriks
# Problem buatlah bintang seperti berikut
# let input = 5
# hasilnya
# *
# **
# ***
# ****
# *****

bintang = "*"

for x in range(5):
    print(bintang)
    bintang += bintang[x]

pola = ""

for y in range(5, 0, -1):
    for z in range(y):
        pola += '*'
    pola += '\n'

print(pola)

File: part 6.5/loop5.js
//5. Bintang asteriks
//Problem buatlah bintang seperti berikut
//let input = 5
//hasilnya
//*
//**
//***
//****
//*****

let bintang = "*";

for (let i = 0; i < 5; i++) {
  console.log(bintang);
  bintang += bintang[i];
}

let hasil = "";

for (let j = 5; j > 0; j--) {
  for (let k = 0; k < j; k++) {
    hasil += "*";
  }
  hasil += "\n";
}
console.log(hasil);
